### PR TITLE
Add script to view field names and clarify README

### PR DESCRIPTION
**Pull request by dzcode**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,18 @@ Sample response:
...
+```
+## Developing
+
+For forms that will be rendered with a context, you'll need to know the 
+names of the fillable fields to figure out should go in that context -- 
+you can use the `view-form-fields` dev-src script to see all field names. Someone
+can also label new fields with the pro version of Adobe Acrobat.
+
+```bash
+lein run -m form-works.form-fields some-template.pdf
```
Comment (leilakass) [inline README.md:49]: should this be `lein run -m form-works.view-form-fields some-template.pdf` ?

File: README.md [hunk 2]
```diff
@@ -37,6 +37,18 @@ Sample response:
 ```
 {:status :ok
  :uri "https://aws.amazon.com/output-bucket/28c72fd5-393c-48bc-bdb7-df3eaa867bb4/rendered.pdf"
+```
+## Developing
+
+For forms that will be rendered with a context, you'll need to know the 
+names of the fillable fields to figure out should go in that context --
```
Comment (leilakass) [inline README.md:44]: nit w missing word: "figure out WHAT should go in that context"

File: README.md (hunk 1 above)
Comment (dzcode) [reply README.md:49]: good catch!

File: README.md (hunk 2 above)
Comment (dzcode) [reply README.md:44]: thanks!

**Pull request merged**